assemble/pom.xml

```
@@ -520,6 +520,16 @@
               </arguments>
             </configuration>
           </execution>
+          <execution>
+            <id>create-jshell</id>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+            <phase>compile</phase>
+            <configuration>
+              <executable>${basedir}/conf/create-jshell.sh</executable>
```

Since this file should not be included as part of the distribution tarball, but is instead part of the build tools, this `create-jshell.sh` file should live in `assemble/src/main/scripts/`

```suggestion
              <executable>src/main/scripts/create-jshell.sh</executable>
```